leetcode/README.md

```diff
@@ -39,17 +39,6 @@ If you have a solution to any of these problems (which are not being [**repeated
 4. Doxygen documentation isn't used in LeetCode solutions. Simple/small documentation or comments should be fine.
 5. Don't include libraries/headers such as `stdio.h`. Your file should be the solution to the problem only.
 
-### 📜 Adding your new solution to the list 📜
```

Comment:
We should probably mention that it is no longer required to do this, but still keep it in there just in case. What do you think?

Reply:
yes, good idea!

Reply:
Feel free to re-add the section again and add a note. We can always improve it later on. :)

Reply:
Fixed :)